app/efs.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_efs_file_system" "wp_efs" {
+    creation_token = "${var.service}-wp-${var.environment}-efs"
+
+    tags = {
+        Name = "${var.service}-wp-${var.environment}-efs"
+        Service = var.service
+        Environment = var.environment
+        Terraform = "true"
+    }
+}
+
+resource "aws_efs_mount_target" "efs_private_a" {
```

Comment:
The mount target can now be looped with for_each.
See my comment for asg.tf

Reply:
I would only use a loop where there is a possibility of adding or removing to a list. In this scenario there is only two and that wouldn't change.